hack/.structured_logging

```diff
@@ -0,0 +1,17 @@
+pkg/kubelet/dockershim/
+pkg/kubelet/dockershim/cm
+pkg/kubelet/dockershim/libdocker
+pkg/kubelet/dockershim/libdocker/testing
+pkg/kubelet/dockershim/metrics
+pkg/kubelet/dockershim/network
+pkg/kubelet/dockershim/network/cni
+pkg/kubelet/dockershim/network/cni/testing
+pkg/kubelet/dockershim/network/hairpin
+pkg/kubelet/dockershim/network/hostport
+pkg/kubelet/dockershim/network/kubenet
+pkg/kubelet/dockershim/network/metrics
+pkg/kubelet/dockershim/network/testing
+pkg/kubelet/dockershim/remote
+pkg/kubelet/server/metrics
+pkg/kubelet/server/stats
+pkg/kubelet/server/stats/testing
```

Comment:
Nit: is this smart enough to do a reduced list of

```suggestion
pkg/kubelet/dockershim
pkg/kubelet/server
```

or are there still missing files in those directories?

Reply:
nope, see the discussion  https://github.com/kubernetes/kubernetes/pull/99090#discussion_r587043479
I included only top level directories initially `pkg/kubelet/dockershim`, `pkg/kubelet/server` but that can cause confusion as which level directory we should add/we shouldn't and also @BenTheElder suggested we follow this for rest of the scripts.

Reply:
edit: pkg/kubelet/server/server.go isn't migrated so we'd still need to keep the last three

I think this is much more difficult to grok on a dir by dir basis like this, particularly when we've also included all of the subdirectories. But it seems that the script doesn't actually work recursively on the subdirectories, so this would be required. So this is fine for now.